bot/src/interactions/commands/account_info.ts

```diff
@@ -50,9 +50,12 @@ export default {
                 { name: '\u200B', value: '\u200B' },
                 { name: 'Steam ID', value: userInDb.steamId ? userInDb.steamId : '*Account not linked*', inline: true },
                 { name: 'Steam profile', value: userInDb.steamId ? `https://steamcommunity.com/profiles/${userInDb.steamId}` : '*Account not linked*', inline: true },
+                { name: '\u200B', value: '\u200B' },
+                //its for checking their old steam names etc.
```

Comment:
Nice ! 
You can remove this comment though and format code please :D

Reply:
Sure